docs/example_n2hp.rst

```diff
@@ -20,16 +20,15 @@ Example hyperfine line fitting for the N\ :sub:`2`\ H+ 1-0 line.
     # background will not automatically be fit 
     # 4 is the number of parameters in the model (excitation temperature,
     # optical depth, line center, and line width)
-    sp.Registry.add_fitter('n2hp_vtau', pyspeckit.models.n2hp.n2hp_vtau_fitter,
-        4, multisingle='multi')
+    sp.Registry.add_fitter('n2hp_vtau', pyspeckit.models.n2hp.n2hp_vtau_fitter,4)
 
     # Run the fitter
-    sp.specfit(fittype='n2hp_vtau',multifit=True,guesses=[15,2,4,0.2])
+    sp.specfit(fittype='n2hp_vtau',multifit=None,guesses=[15,2,4,0.2])
```

Comment:
In these examples, we should be able to entirely remove the `multifit` keyword, since it just adds confusion.